lib/cli.js

```diff
@@ -54,17 +57,17 @@ exports._parse = function (argv, _console, callback) {
     }
 
     // Otherwise, perform our validation
-    async.eachOfLimit(filepathContentArr, program.concurrency,
+    async.eachOfLimit(filepathContentArr, opts.concurrency,
```

Comment:
Was confused why we needed this change -- turns out in Commander@7 they moved it to an inner object to avoid unsafe usage:

https://github.com/tj/commander.js/pull/1409

https://github.com/tj/commander.js/blob/4aaaa9de3fd09401afe3894483193ba0e2e512d7/CHANGELOG.md#700-2021-01-15